staging/src/k8s.io/client-go/util/workqueue/default_rate_limiters.go

```diff
@@ -236,3 +236,42 @@ func (w WithMaxWaitRateLimiter) Forget(item interface{}) {
 func (w WithMaxWaitRateLimiter) NumRequeues(item interface{}) int {
 	return w.limiter.NumRequeues(item)
 }
+
+var _ RateLimiter = &FakeRateLimiter{}
```

Comment:
Let's move these fakes to default_rate_limiters_test.go

Reply:
Move to default_rate_limiters_test.go file